.travis.yml

```diff
@@ -8,6 +8,14 @@ cache:
         - $HOME/.cache/pip
         - $HOME/ossl-1/
 
+# Only build master, the version branches (e.g. 1.7.x), and 
+# version tags (which are apparently considered branches by travis)
+branches:
+    only:
+        - master
+        - /\d+\.\d+\.+x/
+        - /\d+\.\d+\.*\d*/
```

Comment:
The last `.` should be `\.?`, not `\.*`, we don't allow arbitrary numbers, we allow one or zero. Arguable it should be `(\.\d+)?`, since if you have a dot you need some numbers.